deploy/foreman-tasks.sysconfig

```diff
@@ -6,3 +6,15 @@ FOREMAN_LOGGING=warn
 FOREMAN_LOGGING_SQL=warn
 FOREMAN_TASK_PARAMS="-p foreman"
 FOREMAN_LOG_DIR=/var/log/foreman
+
+#Set the number of executors you want to run
+#EXECUTORS_COUNT=1
+
+#Set memory limit for executor process, before it's restarted automatically
+#EXECUTOR_MEMORY_LIMIT=1gb
+
+#Set delay before first memory polling to let executor initialize (in sec)
+#EXECUTOR_INIT_DELAY=180
+
+#Set memory polling interval, process memory will be checked every N seconds.
+#EXECUTOR_POLLING_INTERVAL=60
```

Comment:
Short polling interval is there to prevent bursts from consuming too much memory. For example upload_package_info task that is executed by multiple hosts at once could cause a very rapid memory growth. Since we don't have throttling policy, it would be better to remove overgrown processes before they consume too much memory.

> Katello with all plugins (Satellite) boot time is quite long

We are talking about the executor, not about the whole katello, the executor does not require initialization that is so long. Also, if the process reaches its memory limit in first 180 seconds, it's either the limit is wrong or there is amemory hogging task that needs to get out of the memory sooner than later.